common/notification/index.ts

```diff
@@ -406,6 +410,11 @@ export async function actionTakenAt<ID extends ActionID>(
     noDuplicates = false,
     attachments?: AttachmentGroup
 ) {
+    if (SILENCE_NOTIFICATION_SYTEM) {
```

Comment:
maybe just to be sure ?
```suggestion
    if (SILENCE_NOTIFICATION_SYTEM && isNotProdEnv) {
```

Reply:
good idea